Symfony/CS/Tests/DocBlock/TagTest.php

```diff
@@ -37,6 +37,7 @@ public function provideNameCases()
             array('THROWSSS', "\t@THROWSSS\t  \t RUNTIMEEEEeXCEPTION\t\t\t\t\t\t\t\n\n\n"),
             array('other', ' *   @\Foo\Bar(baz = 123)'),
             array('expectedException', '     * @expectedException Exception'),
+            array('property-read', ' * @property-read integer $daysInMonth number of days in the given month'),
```

Comment:
we should still be able to still return the correct tag with the getName method

although it would technically not change the operation of our fixers, it's still wrong, because `getName()` is returning `other` instead of `foo-bar` or whatever